appserver/extras/docker-images/server-full/src/main/docker/Dockerfile

```diff
@@ -29,25 +29,25 @@ ENV PATH="${PATH}:${PAYARA_DIR}/bin"
 ARG TINI_VERSION=v0.18.0
 
 # Download tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini \
-    https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
```

Comment:
Unless there is special reason for that, we should keep single add and thus single intermediate image.

Reply:
We use --no-cache, so layers are not cached and used.
And also docker does not mention this as allowed in documentation, I found only a merged PR where it was implemented somehow. So I would rather avoid it.
The RUN command is different case, it is in all examples and each execution can make many changes.
https://stackoverflow.com/questions/35594987/how-to-force-docker-for-a-clean-build-of-an-image
https://github.com/fabric8io/docker-maven-plugin/issues/348

Reply:
https://docs.docker.com/engine/reference/builder/#add

`ADD [--chown=<user>:<group>] <src>... <dest>`

Note the ellipsis.

Reply:
OK, you win. I don't know why my brain ignored it :D